hash/hash_blake2b.c

```diff
@@ -0,0 +1,343 @@
+/**
+ * @addtogroup hash Hash algorithms
+ * @{
+ * @file hash_blake2b.c
+ * @author [Daniel Murrow](https://github.com/dsmurrow)
+ * @brief [Blake2b cryptographic hash
```

Comment:
Could you please provide a detailed description of the algorithm?
You can take references from Wikipedia or any other viable/trustworthy source.